src/Ng/Jobs/LoopingNgJob.cs

```diff
@@ -38,7 +38,14 @@ protected LoopingNgJob(ILoggerFactory loggerFactory)
                 }
 
                 Logger.LogInformation("Running job.");
-                await RunInternal(cancellationToken);
+                try
+                {
+                    await RunInternal(cancellationToken);
+                }
+                catch(TransientException te)
```

Comment:
Not possible--`RunInternal` is overridden by every looping job with its functionality. Additionally, the code changes apply to two jobs: Feed2Catalog and Package2Catalog.